docs/runner.rst

```diff
@@ -130,6 +143,73 @@ Common options:
     Server identity used in the 'Server' header in responses. Default
     is 'waitress'.
 
+``--trusted-proxy=IP``
+    IP address of a remote peer allowed to override various WSGI environment
+    variables using proxy headers.
+
+    For unix sockets, set this value to ``localhost`` instead of an IP address.
+
+    The value ``*`` (wildcard) may be used to signify that all remote peers are
+    to be trusted.
+
+``--trusted-proxy-count=INT``
+    How many proxies we trust when chained. For example,
+
+    ``X-Forwarded-For: 192.0.2.1, "[2001:db8::1]"``
+
+    or
+
+    ``Forwarded: for=192.0.2.1, For="[2001:db8::1]"``
+
+    means there were (potentially), two proxies involved. If we know there is
+    only 1 valid proxy, then that initial IP address "192.0.2.1" is not trusted
+    and we completely ignore it.
+
+    If there are two trusted proxies in the path, this value should be set to
+    2. If there are more proxies, this value should be set higher.
+
+    Default: ``1``
+
+``--trusted_proxy_headers=LIST``
```

Comment:
Is it correct that the option uses underscores, not hyphens?

Reply:
There is indeed a mistake. I noticed it myself earlier, but I was working on something else, so haven't submitted a fix yet. If you want to submit the fix, go for it!

Reply:
#458 has the fix.